assets/helpers/stripeCheckout/stripeCheckout.js

```diff
@@ -39,11 +39,25 @@ export const setupStripeCheckout = (
   };
   const defaultCloseHandler: Function = () => {};
 
+  let stripeKey = null;
+
+  switch (currency) {
+    case 'AUD':
+      stripeKey = isTestUser ?
+        window.guardian.stripeKeyAustralia.uat : window.guardian.stripeKeyAustralia.default;
+      break;
+    default:
+      stripeKey = isTestUser ?
+        window.guardian.stripeKeyRestOfTheWorld.uat :
+        window.guardian.stripeKeyRestOfTheWorld.default;
+      break;
+  }
```

Comment:
Is it possible to put this into a separate function, to reduce the length of `setupStripeCheckout`?